inference-engine/src/plaidml_plugin/ops/bucketize.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "plaidml_ops.hpp"
+#include "plaidml_util.hpp"
+
+#include "ngraph/opsets/opset.hpp"
+#include "ngraph/opsets/opset3.hpp"
+
+#include "plaidml/op/op.h"
+
+using namespace plaidml;  // NOLINT[build/namespaces]
+using ngraph::opset3::Bucketize;
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("Bucketize", [](const Context& ctx) {
+  auto* layer = ngraph::as_type<Bucketize>(ctx.layer);
+  IE_ASSERT(ctx.operands.size() == 2);
+  auto A = ctx.operands.at(0);
+  auto B = ctx.operands.at(1);
+
+  std::vector<int> broadcastShape;
+  std::vector<int> bcast_axes;
```

Comment:
Be consistent with variable naming, either use `snake_case` or `camelCase` but not both.

Reply:
Done